src/SGB_Command_Border.md

```diff
@@ -83,8 +84,18 @@ border defining all unique tiles would have to define this many tiles:
 
 Because the CHR RAM allocated by SGB for border holds only 256 tiles, a full-screen border must repeat at least 281 tiles and a letterboxed border at least 90.
 
+It turns out that 29 rows of the border tilemap sent through PCT_TRN are at least partly visible in some situations.
+The SGB system software sets the border layer's vertical scroll position (BG1VOFS) to 0.
+Because the S-PPU normally displays lines BGxVOFS+1 through BGxVOFS+224 of each layer, this hides the first scanline of the top row of tiles and adds one scanline of the nominally invisible 29th row at the bottom.
+Most of the time, SGB hides this extra line with forced blanking (writing $80 to INIDISP at address $012100).
+While SGB is busy doing some things, such as fading out the border's palette, it neglects to force blanking, making the line flicker on some TVs.
```

Comment:
"busy doing some things" doesn't really feel right here. Are we sure we need such phrasing?

Reply:
I don't know the official name of the "some things" that cause SGB to skip copying the GB buffer to VRAM.

Reply:
In SNESdev, zlago suggested "busy processing (some) game or user commands". oziphanto called it "impossible to define what a word would be unless we actually know what 'some things' explicitly are", and finding what does and doesn't cause the system software to show a busy line would be a separate research project in itself.

Reply:
The latest commit has a different wording for this.